notebooks/1_BayesianWorkflow/1_Ins_BayesRefresher.ipynb

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
<p>I'm ok if you find this helpful, but textbooks like Bishop or Goodfellow derive loss functions from a likelihood, so it feels backwards to me!</p><p><br></p><p>"Likelihood can roughly be thought of as loss functions from "Machine Learning""</p>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/bayesian-model-evaluation/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Hmm, I do find it helpful but I also see your point. I tried updating it to be more "correct"